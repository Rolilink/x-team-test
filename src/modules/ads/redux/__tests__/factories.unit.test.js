import adsStateFactory from '../factories';

const stateWithAds = {
  ads: [1, 2],
};

describe('Ads.Redux.Factories.adsState', () => {
  it(
    'should return an state with an empty ads array as default',
    () => (expect(adsStateFactory().ads).toMatchSnapshot()),
  );

  it(
    'should return a populated ads array when an object with ads is received as parameter',
    () => (expect(adsStateFactory(stateWithAds).ads).toEqual(stateWithAds.ads)),
  );
});
