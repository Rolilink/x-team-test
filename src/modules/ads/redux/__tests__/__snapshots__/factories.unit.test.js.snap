// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ads.Redux.Factories.adsState should return an state with an empty ads array as default 1`] = `Array []`;
